platform/core/src/defaults/hotkeyBindings.js

```diff
@@ -173,30 +173,31 @@ const bindings = [
     label: 'W/L Preset 5',
     keys: ['5'],
   },
-  {
```

Comment:
These were causing exceptions when testing because I thought there were keyboard shortcuts to apply the colormap.  There is no actual behavioural removal/change here as none of these worked and were throwing exceptions so it just doesn't do anything now rather than corrupting the images.